Makefile

```diff
@@ -138,6 +138,15 @@ libvirt-share: ## Configure ACLs to allow RWX for libvirt VM (e.g. Admin Worksta
 	@find "$(PWD)" -type d -and -user $$USER -exec setfacl -m u:libvirt-qemu:rwx {} +
 	@find "$(PWD)" -type f -and -user $$USER -exec setfacl -m u:libvirt-qemu:rw {} +
 
+.PHONY: vagrant_package
+vagrant_package: ## Package up a vagrant box of the last stable SD release
```

Comment:
Nit: can we call this `vagrant-package`? It's more consistent with the existing names for makefile targets:

```
 $ make help | perl -lanE 'print $F[0]' | grep '[-_]' -o | sort | uniq -c
      1 _
     19 -
```

I'll ask for similar stylistic changes throughout.